src/components/RestaurantIcon/RestaurantIcon.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './RestaurantIcon.css';
+
+const cuisineIconsMap = {
+  'American': 'R',
```

Comment:
There is a wired coupling between this list and the server list,
I think it's better if it'll sit in the DB too (as part of the metadata maybe ?)